cmd/notary/keys.go

```diff
@@ -90,7 +93,15 @@ type keyCommander struct {
 func (k *keyCommander) GetCommand() *cobra.Command {
 	cmd := cmdKeyTemplate.ToCommand(nil)
 	cmd.AddCommand(cmdKeyListTemplate.ToCommand(k.keysList))
-	cmd.AddCommand(cmdKeyGenerateRootKeyTemplate.ToCommand(k.keysGenerateRootKey))
+	cmdGenerate := cmdKeyGenerateKeyTemplate.ToCommand(k.keysGenerate)
+	cmdGenerate.Flags().StringVarP(
+		&k.outFile,
```

Comment:
Should `--output` only output the public key? Or at least have different flags for pub/priv out?

I'm just thinking about the case where you're storing in a yubikey but still want the public part output to a file.